src/plugins/MiradorImageTools.js

```diff
@@ -277,41 +252,46 @@ class MiradorImageTools extends Component {
             <ToolContainer>
               <MiradorMenuButton
                 aria-label={t('revert')}
-                onClick={this.handleReset}
+                onClick={handleReset}
               >
                 <ReplaySharpIcon />
               </MiradorMenuButton>
             </ToolContainer>
           </React.Fragment>
-          )}
-          {!isSmallDisplay && toggleButton}
-        </Root>
-      </SizeContainer>
-    );
-  }
-}
+        )}
+        {!isSmallDisplay && toggleButton}
+      </Root>
+    </SizeContainer>
+  );
+});
 
 MiradorImageTools.propTypes = {
   enabled: PropTypes.bool,
   open: PropTypes.bool,
-  size: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+  size: PropTypes.shape({
+    width: PropTypes.number,
+    height: PropTypes.number,
+  }).isRequired,
   t: PropTypes.func.isRequired,
   updateViewport: PropTypes.func.isRequired,
   updateWindow: PropTypes.func.isRequired,
   viewer: PropTypes.object, // eslint-disable-line react/forbid-prop-types
   viewConfig: PropTypes.object, // eslint-disable-line react/forbid-prop-types
   windowId: PropTypes.string.isRequired,
+  innerRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.any,
+  ]).isRequired,
```

Comment:
... same here: could you move `innerRef` up to have an alphabetical order?

Reply:
yes sure :)